### PR TITLE
Adding a new section

### PR DESCRIPTION
**Pull request by xdev-user**
I've added a new section to the `file1.md` file.  Take a look at it and tell me what you think!

### CODE REVIEW

File: file1.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # This is a markdown file!
 
 This file has some contents.
+
+## This is a section!
```
Comment (kmpaul) [inline file1.md:5]: I don't like this section title.  Could you please change it to:

```suggestion
## This is a better section title
```

File: file1.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+And it has contents, too!
```
Comment (kmpaul) [inline file1.md:7]: These contents look great!  Thanks, @xdev-user!

**Review by kmpaul — CHANGES_REQUESTED** [review]
Thanks, @xdev-user, for the contribution!  I've made some suggestions for changes.  Please take a look at them and get back to me.

File: file1.md (hunk 2 above)
Comment (xdev-user) [reply file1.md:7]: Thanks!

**Pull request merged**